documentation/RFC/RFC0002-SubscriptionList-in-Get.md

```diff
@@ -0,0 +1,56 @@
+---
+RFC: RFC0002
+Author: Azure PowerShell team
+Status:  Draft
+SupercededBy: n/a
+Version: 1.0
+---
+
+# List of subscriptions for GET commands
+
+A GET cmdlet is mostly used to obtain an Azure resource or validate its existence in Azure.
+With more and more customers using multiple subscriptions the ability to search for resources accross subscriptions has become a more needed convenience.
+
+We are planning to run GET accross subscriptions within a tenant.
+
+## Motivation
+
+Explain the benefits of the change for the users.
+
+```code
+As a PowerShell script developer
+I can run a GET command accross several subscriptions.
+so that I do not have to maintain a list of my suscriptions
+```
+
+## User Experience
+
+The default behavior will search accross all the subscriptions you have access:
```

Comment:
```suggestion
The default behavior will not change, but the user can specify a different subscription, or multiple subscriptions when performing gats.  The output of these gets can be filtered and piped into any other cmdlet.
```

Reply:
@markcowl - the default behaviour should ideally be a search across all available subscriptions unless filtered to current subscription.

However a potential middle ground would be to add a switch parameter `-AllSubscriptions` that allows the search to run across all subscriptions.

Reply:
Yes, this is currently about what preview 1 will do.  We made minimal changes in Az.Accounts for this preview - we will be able to do more in preview 2